### ISSUE TITLE
No suggestion options for enum types if companion object exists on enum class

### ISSUE THREAD

**Issue by Strum355** [fwcd/KotlinLanguageServer#63]
```kotlin
public enum class TokenType(val type: String) {
    ILLEGAL("ILLEGAL"),
    EOF("EOF"),
    /* ... */
    RBRACE("{");

    companion object {
        private val tokens: HashMap<String, TokenType> = HashMap()

        init {
            for(token in TokenType.values()) {
                tokens.put(token.type, token)
            }
        }

        public fun get(type: String): TokenType = this.tokens.getOrDefault(type, TokenType.ILLEGAL)
    }
}
```
With this code, if i type `TokenType.`, i get the following options:
![image](https://user-images.githubusercontent.com/18282288/47261019-24bc6000-d4be-11e8-92d9-193d303ffabc.png)

If i comment out the companion object, i get the full list of enum types:
![image](https://user-images.githubusercontent.com/18282288/47261023-34d43f80-d4be-11e8-9eb9-ef8cf0bda416.png)

**Comment by fwcd** [comment]
Fixed by #92.

**Comment by Strum355** [comment]
Great! Thanks @matthewmcgarvey
